client/src/components/layout/Landing.js

```diff
@@ -0,0 +1,66 @@
+import React, { Fragment, useState, Profiler } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { login, register } from '../../actions/auth';
+
+const Landing = ({ login, register }) => {
```

Comment:
alright. to be honest, this is the first react js I've ever seen. wanna give me a break down of this file, either irl or online?

Reply:
Sure, probably easiest IRL sometime. TL;DR, this is a functional component (as opposed to a class component) which returns some JSX (HTML looking stuff) telling the browser how to render it. It's using some local state to track the form inputs and some redux actions which aren't doing much now but will eventually hit endpoints on the API to either login or register.